.circleci/config.yml

```diff
@@ -32,12 +32,12 @@ jobs:
     docker:
       - image: circleci/node:8
     steps:
-      - attach_workspace:
-          at: .
+      - checkout
+      - run: install-npm
+        command: npm install
```

Comment:
Does this work? I thought run was supposed to be either `run: command` or

```
run:
  name: name
  command: command
```

Reply:
Ah I mistyped, should have copied the config from above.